server/setup.go

```diff
@@ -35,3 +47,34 @@ func getDB(ctx context.Context) (persistence.DB, error) {
 
 	return nil, errors.New(`database type not supported`)
 }
+
+func seedNPCs(db persistence.DB) error {
+	npcIDs := []model.PlayerID{interaction.Dumb, interaction.Simple, interaction.Calc}
+	var err error
+	for _, id := range npcIDs {
+		p := model.Player{
+			ID:   id,
+			Name: string(id),
+		}
+		pm := interaction.New(id, interaction.Means{Mode: interaction.NPC})
+		_, err = db.GetInteraction(id)
+		if err == persistence.ErrInteractionNotFound {
+			err = db.SaveInteraction(pm)
+			if err != nil {
+				break
+			}
+		} else {
+			break
+		}
+		_, err = db.GetPlayer(p.ID)
+		if err == persistence.ErrPlayerNotFound {
```

Comment:
same comment about err checking here. and I don't think we should ever break out of this loop...

and then at the last return in the func we can just `return nil`